comps/comps-katello-el9.xml

```diff
@@ -12,12 +12,12 @@
     <packagelist>
       <!-- katello core packages -->
       <packagereq type="default">foreman-proxy-content</packagereq>
-      <packagereq type="default">katello</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello</packagereq> -->
       <packagereq type="default">katello-client-bootstrap</packagereq>
       <packagereq type="default">katello-common</packagereq>
-      <packagereq type="default">katello-debug</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello-debug</packagereq> -->
       <packagereq type="default">katello-repos</packagereq>
-      <packagereq type="default">katello-selinux</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello-selinux</packagereq> -->
```

Comment:
Is this the `candlepin-selinux` dependency?

Reply:
Yes